src/utils/context.ts

```diff
@@ -149,6 +156,31 @@ function makeCloudTraceHeader(): string {
   return `${trace}/${spanId}`;
 }
 
+/**
+ * getContextFromOtelContext looks for the active open telemetry span context
+ * per Open Telemetry specifications for tracing contexts.
+ *
+ * @param projectId
+ */
+export function getContextFromOtelContext(
+  projectId: string
+): CloudTraceContext | null {
+  const spanContext = trace.getActiveSpan()?.spanContext();
+  const FLAG_SAMPLED = 1; // 00000001
+  if (spanContext !== undefined && isSpanContextValid(spanContext)) {
+    const otelSpanContext = {
+      trace: spanContext?.traceId,
+      spanId: spanContext?.spanId,
+      traceSampled: (spanContext.traceFlags & FLAG_SAMPLED) !== 0,
+    };
+
+    const return_context = toCloudTraceContext(otelSpanContext, projectId);
+    return return_context;
```

Comment:
nit could just return directly instead of the assignment

Reply:
Done.